lib/stat_tracker.rb

```diff
@@ -29,9 +29,13 @@ def total_games
     @games.count { |game| game }.to_f
   end
 
-  # def percentage_home_wins
+  def percentage_home_wins
+    home_wins = @game_teams.count do |game|
```

Comment:
While we can refactor this at some point as you mentioned, I think the readability provided here is quite nice so may not be necessary